### PR TITLE
Add typescript to webpack build

### LINKED ISSUE #2195: fix type checking
Add loader `https://github.com/s-panferov/awesome-typescript-loader`
to webpack.common.js

need to fix types.

### PR DESCRIPTION
**Pull request by justinbarry**
Added typings-for-css-modules-loader package to try to prevent style issues in TS.

closes #2195

### CODE REVIEW

File: packages/augur-ui/src/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import accountPositions from "packages/augur-ui/src/modules/positions/reducers/account-positions";
```
Comment (bthaile) [inline packages/augur-ui/src/reducers/index.js]: might as well make this a typescript file and use "modules/...." import statements don't like the full path.

**Comment by bthaile** [comment]
looks good, I guess we'll go with the index.ts files. reducers.js can be renamed to reducers.ts

File: packages/augur-ui/src/reducers/index.js (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/reducers/index.js]: I made the switch in previous PR that got merged to master

File: packages/augur-ui/src/select-state.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const selectPendingLiquidityOrders = state =>
   state.pendingLiquidityOrders;
 export const selectAccountShareBalance = state => state.accountShareBalances;
 export const selectFilledOrders = state => state.filledOrders;
-
+reporting-resolved.ts
```
Comment (andrewdamelio) [inline packages/augur-ui/src/select-state.ts]: a wild ```reporting-resolved.ts appears```

**Review by bthaile — APPROVED** [review]
lgtm, lets get it merged up and start fixing the errors

**Pull request merged**